tests/index.test.js

```diff
@@ -1,80 +1,113 @@
-const error = require('../dist');
```

Comment:
The only changes in this file were moving some utility functions that were only used in this test from the action itself to this test.